installer/bundle/bundle_skel.sh

```diff
@@ -1000,6 +1000,7 @@ case "$installMode" in
         ;;
 
     U)
+        echo "trace 0 installMode U" >&2
```

Comment:
Can you explain why you want to commit these trace lines? They're great for development debugging, but may not be appropriate for public release.